mailbox/store/src/test/resources/eml/nonTextual.json

```diff
@@ -133,7 +133,6 @@
         "meta:table-count":["0"],
         "Creation-Date":["2015-06-18T12:41:25.197399866"],
         "xmpTPg:NPages":["1"],
-        "resourceName":["toto.odt"],
```

Comment:
This header is always added by the embedded Tika, not by the /rmeta endpoint.
When using the Tika as a server, this header is given by the /detect endpoint:
https://wiki.apache.org/tika/TikaJAXRS#Detector_Resource
https://github.com/apache/tika/blob/0d8f030f02830a39e004fbfbd6989128ec2047d7/tika-server/src/main/java/org/apache/tika/server/resource/DetectorResource.java#L51